applications/Polkadot_Common_Good_Apps_Hub.md

```diff
@@ -0,0 +1,307 @@
+
+[Repository Link](https://github.com/Polkadot-Common-Good-Apps/polkadot-sdk#polkadot-common-good-apps)
+
+## Development Roadmap :nut_and_bolt:
+### Overview
+
+- **Total Estimated Duration:** 3 months.
+- **Full-Time Equivalent (FTE):**  3 FTE
+- **Total Costs:** 29,900 USD
+
+### Milestone 1 — Parachain node and runtime setup.
+
+- **Estimated duration:** 1 month
+- **FTE:**  3
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code. |
+| **0c.** | Zombienet | We will provide a zombienet configuration file to spin up the chain locally. |
+| 1. | Runtime | The parachain runtime with its corresponding XCM configuration will be delivered.|
+| 2. | Parachain node | A cumulus-based node containing the runtime chain specs will be developed and provided. |
+| 3. | FRAME Pallets | The NFT, Assets, Proxy and Utility FRAME pallets will be configured. |
+
+### Milestone 2 — Contracts pallet and custom upload pallet
+
+- **Estimated duration:** 1 month
+- **FTE:**  3
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code. |
+| **0c.** | Testing and Testing Guide | The custom contracts upload pallet will contain the corresponding unit testing for assuring code quality. A comprehensive guide to test the milestone will be delivered detailing each step. |
+| 1. | Contracts Pallet | The pallet will be configured in the runtime to allow WASM based smart contracts executions in the parachain. |
+| 2. | Custom contract upload process | According to the contracts upload process described, the corresponding logic for the pallet will be delivered including it's documentation, XCM related work and it's corresponding testing guide |
+
+### Milestone 3 — Pallet Randomness.
+
+- **Estimated duration:** 1 month
+- **FTE:**  3
+- **Costs:** 9,900 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide a user tutorial to visualize the random value generated from the pallet. |
+| 1. | Randmoness Pallet | Develop the specifications and details described in the pallet randomness section. |
+| 2. | Dummy Random Testing Pallet | A testing pallet for reading the random value generated form the randomness pallet. This pallet will test mostly the Randmoness trait implementaion. |
```

Comment:
@keeganquigley:  I have updated this item to reflect the purpose of this testing pallet.